modules/apps/document-library/document-library-web/build.gradle

```diff
@@ -1,4 +1,6 @@
 dependencies {
+	compileInclude group: "com.google.code.gson", name: "gson", version: "2.8.6"
+
```

Comment:
@thienquach87 
remove this when we use JSONFactoryUtil

Reply:
removed